src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -986,6 +985,36 @@ void CommandRequestImpl::AddTimeOutComponentInfoToMessage(
   }
 }
 
+void CommandRequestImpl::AddRequestToTimeoutHandler(
+    smart_objects::SmartObject& request) const {
```

Comment:
@LitvinenkoIra can request be const?

Reply:
@LuxoftAKutsan yes, fixed 278050fc416a577487a1bd2acbf1eedfdbea3c37